feat($theme-default): allow optional subtitle (#1981)
vuejs/vuepress: packages/docs/docs/theme/default-theme-config.md

---
```

> You can disable `title` and `subtitle` by setting the correspoding fields to `null`.

Reviewer: ```suggestion
You can disable `title` or `subtitle` in the Homepage by using `null` value.
```
